examples/units_and_coordinates/map_slit_extraction.py

```diff
@@ -0,0 +1,61 @@
+"""
+=====================================================
+Extracting intensity of a map between two coordinates
+=====================================================
+
+In this example we will define a slit in world coordinates and then extract the intensity values of all the pixels that the path intersects with.
```

Comment:
> a slit in world coordinates

I think this is too jargony (both "slit" and "world coordinates") for the first line of an example description. Since this example isn't about constructing a line (users might have coordinates already from some other analysis), I'd suggest dropping the first part of this sentence.

Reply:
I reworded this slightly